src/components/Attachments/AttachmentView/index.js

```diff
@@ -118,6 +119,7 @@ function AttachmentView({
                 onScaleChanged={onScaleChanged}
                 onToggleKeyboard={onToggleKeyboard}
                 onLoadComplete={() => !loadComplete && setLoadComplete(true)}
+                errorLabelStyles={isUsedInAttachmentModal ? [styles.textLabel, styles.textLarge] : []}
```

Comment:
```suggestion
                errorLabelStyles={isUsedInAttachmentModal && [styles.textLabel, styles.textLarge]}
```

Reply:
This will send a `boolean` down.  Id rather limit the leakage here. What do you think. ?

Reply:
I think it's fine as long as we have a defaultProps, it'll be replaced with `[]`